.rusty-hook.toml

```diff
@@ -0,0 +1,5 @@
+[hooks]
+pre-commit = "forge fmt && forge test -vvv && forge snapshot --check --match-contract Gas"
```

Comment:
i need to revise the last shell command `forge snapshot --check --match-contract Gas`; it won't allow commits with a LOWER gas snapshot